lib/carrierwave/downloader/base.rb

```diff
@@ -40,7 +40,9 @@ def download(url, remote_headers = {})
       def process_uri(uri)
         uri_parts = uri.split('?')
         encoded_uri = Addressable::URI.parse(uri_parts.shift).normalize.to_s
-        encoded_uri << '?' << Addressable::URI.encode(uri_parts.join('?')).gsub('%5B', '[').gsub('%5D', ']') if uri_parts.any?
+        encoded_uri << '?' << Addressable::URI.encode(uri_parts.join('?')).
+          gsub('%5B', '[').gsub('%253A%252F%252F', '%3A%2F%2F').
```

Comment:
This is specifically targeting for the `://` portion of the `https://` or `http://` or `some-protocol://` and removing the double escape.